app/views/templates/_form.html.erb

```diff
@@ -33,11 +33,11 @@
         <%= alert :class => 'alert-warning', :header => '',
                   :text  => icon_text("warning-sign", (_("Warning: This template is locked. You may only change the associations. Please %s it to customize.") %
                                                             link_to(_('clone'), template_hash_for_member(::Template.find(params[:id]), 'clone_template'))).html_safe) %>
-
       <% else -%>
-        <%= alert :class => 'alert-success', :header => '',
-                  :text  => icon_text("info-sign", (_('Note: %s ') % link_to(_('Useful template functions and macros'),
-                                                                             "http://www.theforeman.org/manuals/#{SETTINGS[:version].short}/index.html#4.4.3ProvisioningTemplates", :rel => 'external')).html_safe) %>
+          <%= alert :class => 'alert-success', :header => '',
+                    :text  => icon_text("info-sign", (_('Note: %s ') % link_to(_('Useful template functions and macros'),
+                                                                               type == 'ptable' ? documentation_url("4.4.4PartitionTables") : documentation_url("4.4.3ProvisioningTemplates"),
```

Comment:
I don't know if we still want to display this for non-ptable/ProvisioningTemplate models, perhaps @ares has a view on what would be useful to do for plugins?  I'd suggest just leaving out the entire alert block for unknown types.